packages/stacks-docs/product/components/notices.html

```diff
@@ -21,19 +19,12 @@
     data-controller="s-toast"
     data-s-toast-target="toast"
     data-s-toast-return-element=".js-toast-open[data-target='#launch-toast-base']">
-    <div class="s-notice s-notice__info d-flex ai-center gx16 jc-space-between p8 pl16 wmn4">
-        <p class="m0">
-            Info toast with an undo button.
-        </p>
-        <div class="d-flex">
-            <button type="button" class="s-btn s-notice--btn">Undo</button>
-            <button
-                type="button"
-                class="s-btn s-notice--btn js-toast-close"
-                aria-label="Dismiss"
-                data-action="s-toast#hide">
-                {% icon "ClearSm", "m0" %}
-            </button>
+    <div class="s-notice s-notice__info d-flex ai-center wmn4">
+        <span class="s-notice--icon">{% icon "Info" %}</span>
+        Toast notice message with an undo button
+        <div class="d-flex ml-auto">
+            <button type="button" class="s-link s-link__underlined">Undo</button>
+            <button type="button" class="s-link s-notice--dismiss js-toast-close" aria-label="Dismiss" data-action="s-toast#hide">{% icon "ClearSm" %}</button>
```

Comment:
I opted to style the buttons with `s-link` but not sure if I should've used `s-btn s-btn__link` instead like we do in the Svelte component for Notices.